pandas/_libs/tslibs/dtypes.pyx

```diff
@@ -106,3 +106,20 @@ _period_code_map.update({
     "W": 4000,   # Weekly
     "C": 5000,   # Custom Business Day
 })
+
+
+class FreqGroup:
```

Comment:
Maybe overlooking something simple but do we need both or does the enum alone suffice?

Reply:
the one in the .pxd we can't access from python-space

Reply:
also i expect before long we'll make this one into `class FreqGroup(Enum)` to de-duplicate it with Resolution